### DISCUSSION TITLE
Has the new version of autogenstudio been released yet?

### DISCUSSION THREAD

**Discussion by shanmu-raoyunfei** [microsoft/autogen#4343] [General]
Has the new version of autogenstudio been released yet?The version shown below：

![image](https://github.com/user-attachments/assets/62334b39-baa9-406c-a955-9b7032408c29)

**Comment by shanmu-raoyunfei** [comment]
The version I installed using "pip install autogenstudio" is still the old version.

**Comment by jackgerrits** [comment]
Please see this thread for discussion about the in-development next version.

https://github.com/microsoft/autogen/discussions/4208
